manifests/p/PaddiM8/kalker/2.0.0/PaddiM8.kalker.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: PaddiM8.kalker
+PackageVersion: 2.0.0
+PackageLocale: en-US
+Publisher: PaddiM8
+PackageName: kalker

```

Comment:
```suggestion
Publisher: Kalker
PackageName: Kalker
```
This must match Control Panel:
![image](https://user-images.githubusercontent.com/11600822/183248608-e4e4704c-ee2f-40f4-a6a3-34dcb3982240.png)